qcodes/instrument_drivers/Keysight/keysight_b220x.py

```diff
@@ -0,0 +1,358 @@
+                           val_mapping={True: 1,
+                                        False: 0},
+                           docstring="Param: True for ON, False for OFF"
+                           )
+
+        self.add_parameter(name='gnd_input_port',
+                           get_cmd=':AGND:PORT? {card}'.format(
+                               card=self._card),
+                           set_cmd=':AGND:PORT {card},{{}}'.format(
+                               card=self._card),
+                           vals=MultiType(KeysightB220X._available_input_ports,
+                                          Enum(-1)),
+                           get_parser=int,
+                           docstring="Selects the input that will be used as "
+                                     "GND input port (default 12). The GND "
+                                     "input port cannot be used on subsequent "
+                                     "`connect` or `disconnect` commands if "
+                                     "GND mode is ON"
+                           )
+
+        self.add_parameter(name='gnd_mode',
+                           get_cmd=':AGND? {card}'.format(card=self._card),
+                           set_cmd=':AGND {card},{{}}'.format(card=self._card),
+                           val_mapping={True: 1,
+                                        False: 0}
+                           )
+
+        self.add_parameter(name='unused_inputs',
+                           get_cmd=':AGND:UNUSED? {card}'.format(
+                               card=self._card),
+                           set_cmd=":AGND:UNUSED {card},'{{}}'".format(
+                               card=self._card),
+                           get_parser=lambda response: [int(x) for x in
+                                                        response.strip(
+                                                            "'").split(',') if
+                                                        x.strip().isdigit()],
+                           set_parser=lambda value: str(value).strip('[]'),
+                           vals=Lists(KeysightB220X._available_input_ports)
+                           )
+
+        self.add_parameter(name='couple_ports',
+                           get_cmd=':COUP:PORT? {card}'.format(card=self._card),
+                           set_cmd=":COUP:PORT {card},'{{}}'".format(
+                               card=self._card),
+                           set_parser=lambda value: str(value).strip('[]()'),
+                           get_parser=lambda response: [int(x) for x in
+                                                        response.strip(
+                                                            "'").split(',') if
+                                                        x.strip().isdigit()],
+                           vals=Lists(Enum(1, 3, 5, 7, 9, 11, 13))
+                           )
```

Comment:
`couple_ports` and `unused_inputs` have same `get_parser` and `set_parser`. What about making those reused private methods?